docs/intro/contributing.txt

```diff
@@ -222,7 +222,7 @@ some other flavor of Unix, run:
 
 .. console::
 
-    $ ./runtests.py
+    $ python ./runtests.py
```

Comment:
Just above this it says "If you're using GNU/Linux, macOS, or some other flavor of Unix, run:"
This should be updated if we're making changes for Windows 🙂

Reply:
Well noted 👍
I would include that when making the changes for Windows.